### ISSUE TITLE
The extension isn't showing me coverage generated with nyc

### ISSUE THREAD

**Issue by HughePaul** [markis/vscode-code-coverage#39]
I have converage generated with nyc reporter lcov, but it isn't showing in vscode.
The lcov.info seems to be loaded as it shows in problems, but if i click on one item it tried to load a file in the os root (eg /lib/test.js instead of /home/me/myproject/lib/test.js)

Is the extension expecting full path names in the lcov.info? if it is a relative path can it be resolved with the current open directory. I don't have a workspace set, i only open directories.

**Comment by markis** [comment]
Running `nyc` in the example project (https://github.com/markis/vscode-code-coverage/tree/master/example), it seems to have generated the absolute paths for the files in `lcov.info`.  I think vscode could work with relative paths, however lcov has the absolute paths, so it was easier to work with the absolute paths.

```
TN:
SF:/home/<me>/src/vscode-code-coverage/example/index.ts
FN:1,add
FN:5,subtract
FNF:2
FNH:1
FNDA:1,add
FNDA:0,subtract
DA:1,1
DA:2,1
DA:5,1
DA:6,0
LF:4
LH:3
BRF:0
BRH:0
end_of_record
TN:
SF:/home/<me>/src/vscode-code-coverage/example/test.ts
FNF:0
FNH:0
DA:1,1
DA:2,1
DA:4,1
LF:3
LH:3
BRF:0
BRH:0
end_of_record```

**Comment by gilescope** [comment]
The code assumes absolute paths at the moment, src/lib.rs got translated into /src/lib.rs which wasn't right. I've got a PR #45 which grounds relative paths in the workspace root folder. But if you're already doing fully qualified paths in your .info file then it *should* work...

**Comment by hoekma** [comment]
@markis -- Was this fixed in v1.2?   I've been using code-coverage on a client mac and it works great -- tried to install on my own mac last night and it seems to be suffering from the same fixed pathing issue.    Maybe it's not up in the VSCode extensions store yet?  

BTW -- brilliant and indispensable tool!!!

**Comment by markis** [comment]
Glad you like it and yes it should be fixed in v1.2!  Thanks @gilescope! The issue is that my personal access token to publish the extension isn't working.  I have been emailing the support team at Microsoft to get it fixed.  Hopefully it will be published soon.

**Comment by markis** [comment]
...I spoke too soon, the token situation has been address and it's released now!

**Comment by hoekma** [comment]
@markis Yep, automagically fixed! :). Thanks!!

**Comment by hoekma** [comment]
I think I spoke too soon.  It was working and then stopped working. Not sure what changed.  When I click on one of the files in problems pane, it says it cant' find the file at `/file:Users/hoekma/GitHub/{...path to my file}`.  Not sure if the slash before file is doing it.    

My NYC config looks like `  "nyc": {
    "reporter": [
      "lcov",
      "text-summary"
    ],
    "all": true,
    "include": [
      "src/**/*.js"
    ],
    "exclude": [
      "**/*.unit.spec.js",
      "**/*.integration.spec.js",
      "**/testConfig.js",
      "**/index.js"
    ]
  }`

I've tried removing the code-coverage extension, deleting the ./coverage and ./.nyc_output folders, quitting VS Code and restarting, reinstalling code-coverage, and then re-running my tests to generate the ./coverage/lcov.* files.

**Comment by hoekma** [comment]
@markis  FYI -- In regards to my last comment -- I downgraded nyc to version 14.1.1 and it started working again both with the underlines as well as being able to click files in the problems pane.

**Comment by markis** [comment]
Should be working now that https://github.com/markis/vscode-code-coverage/pull/52 has been released in v1.2.1
